core/examples/luigi-sample-vue/README.md

```diff
@@ -0,0 +1,51 @@
+# Sample luigi application written in Vue.JS
```

Comment:
`Luigi`

Also: this document doesn't have the obligatory Overview section. See the template for the README.md documents:
https://github.com/kyma-project/community/blob/master/guidelines/internal-guidelines/repository-template/template/README.md

Reply:
I would suggest to pull from `remote:master` again, as we have updated the README.md there which you can use as your starting point.